package/snapper.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed Dec  6 16:49:06 CET 2017 - kukuk@suse.de
+
+- Add missing systemd services/timer macros
+- Remove cron scripts
+- Workaround systemd preset due to missing systemd macros before
+- 0.5.3
+
```

Comment:
Fate number is missing for our processes.

Reply:
For adapting to upstream changes fate/bsc# is not needed (even though I agree that this might deserve an RN entry).